runtime/src/pallet_assets_mock.rs

```diff
@@ -0,0 +1,190 @@
+// Mock implementation for Assets that satisfies token_gateway requirements
```

Comment:
It seems to me that this mock in never actually used in the current configuration, not even for the benchmarks.

If this is correct, I would suggest replacing all the random implementations of this pallet with something like `unimplemented!`, with some sensible message to make it clear that it is not supposed to be used.